### ISSUE TITLE
Allow enum to output string initializers (TypeScript 2.4)

### ISSUE THREAD

**Issue by PandaWood** [reinforced/Reinforced.Typings#69]
Would be good if this were a setting in an attribute
eg 
```
[TsEnum(UseString = true)]
public enum SomeType {
  Done,
  InProgress
}
```

Output eg  `export enum SomeType { Done = "Done", InProgress = "InProgress" }`

**Comment by pavel-b-novikov** [comment]
Fixed in version 1.4.93
[Unit test](https://github.com/reinforced/Reinforced.Typings/blob/master/Reinforced.Typings.Tests/SpecificCases/SpecificTestCases.EnumStringInitializers.cs)

**Comment by pavel-b-novikov** [comment]
@SylvainFI [Here](https://github.com/reinforced/Reinforced.Typings/blob/ef3aaf3713affd3ec277c705548f39074122b519/Reinforced.Typings.Tests/SpecificCases/SpecificTestCases.EnumStringInitializers.cs#L32)
